openssl-dynamic/src/main/c/tcn.h

```diff
@@ -106,85 +104,37 @@ jstring         tcn_new_stringn(JNIEnv *, const char *, size_t);
 #define AJP_TO_JSTRING(V)   (*e)->NewStringUTF((e), (V))
 
 #define TCN_CHECK_NULL(V, M, R)                      \
-    TCN_BEGIN_MACRO                                  \
+    NETTY_JNI_UTIL_BEGIN_MACRO                       \
         if (V == NULL) {                             \
             tcn_ThrowNullPointerException(e, #M);    \
             return R;                                \
```

Comment:
preexisting: macros which prematurely return behind the scenes increase the risk of memory leak (e.g. what if something is allocated before this point). the risk might be low for null check of input params, but can lead to subtle bugs.

Reply:
Will check as a followup